react-js/react.md

```diff
@@ -41,29 +42,108 @@ You will spend the day working through these materials.
 
 ### Topics to learn & Facilitate
 
-- What is React?
-    - React is a library for creating User Interfaces.
-    - This means it is used for creating what some refer to as the "view layer" of an application.
-- JSX adds to our JavaScript to allow for HTML-like markup in JavaScript.
-- Show Hello World example, plus expanded example.
-- Example for appending to a list in jQuery vs. the same in React.
+- React lets you generate HTML using JavaScript
+- The HTML can change over time, unlike regular HTML
+- No jQuery needed
+
+### Examples
+- Hello World component
+```
+helloWorld.js
+
+import ReactDOM from 'react-dom'
+
+const HelloWorld = () => <h1>Hello, World!</h1>
+      
+ReactDOM.render(<HelloWorld />, document.getElementById('app'))
+
+index.html (The rendering of the above React code will be done in this file)
+
+<div id = "app"></div>
+
+```
+
+- Component with a single prop
+``` 
+welcome.js
+
+import ReactDOM from 'react-dom'
+
+function Welcome(props) {  
+return <h1>Hello, {props.name}</h1>;
+}
+
+const element = <Welcome name="ABC" />;
+ReactDOM.render(
+  element,
+  document.getElementById('root')
+);
+
+index.html (The rendering of the above React code will be done in this file)
 
+<div id = "root"> </div>
+
+```
+
+- Component assigning a `className`
+```
+// ParentComponent.js
```

Comment:
This is way too complicated out of the gate. They don't even know what "a component" is, let alone a hierarchy. Please add the simplest possible hello world in a codepen and move this to the later part of the lesson

Reply:
Ok. Changed to an easier example now.